### PR TITLE
util: extend util module.

### PR DESCRIPTION
**Pull request by cristiantpop**
Extend module with functions and macros used in "tinyiiod" project.

Signed-off-by: Cristian Pop <cristian.pop@analog.com>

### CODE REVIEW

File: projects/drivers/util/util.h [hunk 1]
```diff
@@ -121,10 +125,17 @@ uint32_t hweight8(uint32_t word);
...
+uint64_t div64_u64_rem(uint64_t dividend, uint64_t divisor,
+		       uint64_t *remainder);
 /* Unsigned 64bit divide with 32bit divisor with remainder */
 uint64_t div_u64_rem(uint64_t dividend, uint32_t divisor, uint32_t *remainder);
 /* Unsigned 64bit divide with 32bit divisor */
 uint64_t div_u64(uint64_t dividend, uint32_t divisor);
+/* Verify if str1 equals str2 */
+bool strequal(const char *str1, const char *str2);
+/* Converts from string to int32_t */
+int32_t read_value(const char *str);
```
Comment (dbogdan) [inline projects/drivers/util/util.h]: Let's use a more explicit name for this function.

**Comment by cristiantpop** [comment]
V2:

- rename functions to: "str_to_int32" and "srt_to_uint32".

File: projects/drivers/util/util.c [hunk 2]
```diff
@@ -188,3 +190,46 @@ uint64_t div_u64(uint64_t dividend, uint32_t divisor)
...
 	return div_u64_rem(dividend, divisor, &remainder);
 }
+
+/**
+ * Compare two strings
+ * @param *str1 pointer to string 1
+ * @param *str2 pointer to string 2
+ * @return TRUE if strings are equal, 0 otherwise
+ */
+bool strequal(const char *str1, const char *str2)
```
Comment (dbogdan) [inline projects/drivers/util/util.c]: I would not create a new function that's only a !strcmp()... but there might be a good reason.

Comment (cristiantpop) [reply projects/drivers/util/util.c]: "strequal" was in the initial version of the project. "!strcmp()" will work fine, I will change that.

**Comment by cristiantpop** [comment]
V3;

- removed "strequal".

**Comment by cristiantpop** [comment]
V4: 

- add separate commit for astyle.

**Pull request merged**